utils/src/funTest/kotlin/storage/HttpFileStorageTest.kt

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (C) 2019 HERE Europe B.V.
+ *
```

Comment:
This uses parts of `scanner/src/funTest/kotlin/storages/ArtifactoryStorageTest.kt` so please keep "Copyright (C) 2019 Bosch Software Innovations GmbH".

Reply:
Done.